server/setup.go

```diff
@@ -1,15 +1,31 @@
 package server
 
 import (
+	"github.com/joshprzybyszewski/cribbage/server/interaction"
 	"github.com/joshprzybyszewski/cribbage/server/persistence"
 	"github.com/joshprzybyszewski/cribbage/server/persistence/memory"
 )
 
+// Setup connects to a database and starts serving requests
 func Setup() error {
 	cs := cribbageServer{
 		db: getDB(),
 	}
 
+	npcTypes := []interaction.NPCType{
+		interaction.Dumb, interaction.Simple, interaction.Calculated,
+	}
+	for _, npcType := range npcTypes {
+		npc := interaction.NewNPCPlayer(npcType, cs.handleAction)
+		if _, err := cs.db.GetInteraction(npc.ID()); err != nil {
+			// TODO we should probably not use the error this way...
+			if err.Error() == `does not have player` {
```

Comment:
we can use equality on
https://github.com/joshprzybyszewski/cribbage/pull/17/files#diff-2c5c023d4d8459dbc4aee14fc7bd1f1eR24

when that gets in